packages/importer/src/importer.ts

```diff
@@ -221,6 +221,14 @@ export class OneSchemaImporterClass extends EventEmitter {
       }
     }
 
+    if (
```

Comment:
hmm: is there actually no better way to set the default for this?.. I don't think its a big deal either way tho

Reply:
i could abbreviate with the ? operator, but don't really feel like it's worth it.. could also object spread, but I think this is clearer.. if you prefer one of those methods, i don't mind changing it